### PR TITLE
Merge pull request #4 from ijkilchenko/master

### PR DESCRIPTION
**Pull request by Ztilocco**
Added a few more layers

### CODE REVIEW

**Comment by Ztilocco** [comment]
Finished test and passed by reference.

**Comment by Ztilocco** [comment]
Finished test.

File: convolutional_neural_network.cpp [hunk 1]
```diff
@@ -36,14 +37,14 @@ class Layer {
   }
 
   vector<vector<double>> static add_matrices(vector<vector<double>> a, vector<vector<double>> b) {
-    vector<vector<double>> c;
+    
+    vector<vector<double>> c(a.size(), vector<double>(a[0].size(), 0));
```
Comment (ijkilchenko) [inline convolutional_neural_network.cpp:41]: Nice! This is faster!

File: convolutional_neural_network.cpp [hunk 2]
```diff
@@ -562,7 +586,7 @@ int main() {
   // Do a forward pass with the first "image"
   // model.h(X[1]);
 
-  cout << "Tests finished!\n";
+  cout << "Tests finished!!!!\n";
```
Comment (ijkilchenko) [inline convolutional_neural_network.cpp:589]: 😃

**Review by ijkilchenko — APPROVED** [review]

**Pull request merged**